docs/template.asciidoc

```diff
@@ -0,0 +1,62 @@
+[[page-title]]
+= Page title
+
+[partintro]
+--
+
+Include a short description. The short description should always include 
+the value prop, answer why users want to use the feature, and how the 
+feature benefits them. 
+
+For easy navigation, include the links to the key sections in a bulleted 
+list. This way, users can click exactly where they want to go instead of 
+scrolling down the page:
+
+* <<task-section-description, Task section title>>
+* <<section2-description, Task section title>>
+* <<section3-description, Task section title>>
+
+Provide a screenshot or video.
+
+TIP: Need inspiration? Here are some good examples:
+* <<snapshot-repositories, Snapshot and Restore>>
+* <<dashboard, Dashboard>>
+* <<canvas, Canvas>>
+
+[float]
+[[concept-section-description]]
+== Concept section title
+
+Include content that further describes the feature and its capabilities.
+
+--
+
+[[task-section-description]]
+== Task section title
+
+Provide an example or walkthrough of how to use the feature. Whenever possible, 
+use the <<add-sample-data, sample data>>.
+
+[float]
+[[section2-description]]
+=== Task section title
+
+To provide steps, use the following:
+
+. Step 1.
+. Step 2.
+. Step 3.
+. Step 4.
++
+[role="screenshot"]
+image:images/example_screenshot.png[Helpful image]
+
+. Step 5.
+
+TIP: When you create task sections, you only need to explain the details that users can't see on the screen. Avoid giving users a tour of the UI or providing instructions on how to use every action on the screen. Delete and save are common actions that users can typically figure out themselves.
```

Comment:
Move this content under Task section title.   

We only need to include steps for complex procedures.  If you're just repeating the text in the UI, then you don't need to write the task